src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_CallerInfoAttributes.cs

```diff
@@ -64,22 +64,28 @@ public static void Main()
 }
 ";
 
-            var compilation = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular10);
+            var compilation = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.ReleaseExe, parseOptions: TestOptions.RegularPreview);
```

Comment:
>RegularPreview [](http://example.com/codeflow?start=160&length=14)

Instead of doing this change I suggest reverting all change to this test, including a change to expected IL. And I suggest doing the same for all other tests affected the same way. I.e. explicitly target the language version for them that doesn't support the caching. There are several advantages in this strategy. Less tests to patch and review. Extra validation that the feature is disabled as we expect it to be.